src/main/scala/io/sdkman/changelogs/package.scala

```diff
@@ -1,26 +1,29 @@
 package io.sdkman
 
 import com.mongodb.client.MongoDatabase
-import com.mongodb.client.model.{Filters, Updates}
+import com.mongodb.client.model.{ Filters, Updates }
 import org.bson.Document
 
 import scala.language.implicitConversions
 
 import scala.collection.JavaConverters._
+import scala.collection.immutable.List
 
 package object changelogs {
 
-  case class Candidate(candidate: String,
-                       name: String,
-                       description: String,
-                       default: String,
-                       websiteUrl: String,
-                       distribution: String)
+  case class Candidate(
+    candidate:    String,
+    name:         String,
+    description:  String,
+    default:      String,
+    websiteUrl:   String,
+    distribution: String)
 
-  case class CandidateVersion(candidate: String,
-                              version: String,
-                              platform: Option[String],
-                              url: String)
+  case class CandidateVersion(
+    candidate: String,
+    version:   String,
+    platform:  Option[String],
+    url:       String)
```

Comment:
Reformatting not a part of this PR. Also, once again not standard scala code style.